initrd/init

```diff
@@ -28,6 +28,11 @@ if ! unsealtotp.sh ; then
 fi
 echo
 
+# Load the network modules and get an IP address
```

Comment:
This should go into a separate MOC startup script that is specified in `CONFIG_BOOTSCRIPT`-- `initrd/init` is only for things that are common to all platforms.  See https://github.com/osresearch/heads/blob/master/config/x230-qubes.config#L21 and  `initrd/bin/qubes-init` for an example